go.sum

```diff
@@ -34,6 +34,8 @@ github.com/konsorten/go-windows-terminal-sequences v1.0.3 h1:CE8S1cTafDpPvMhIxNJ
 github.com/konsorten/go-windows-terminal-sequences v1.0.3/go.mod h1:T0+1ngSBFLxvqU3pZ+m/2kptfBszLMUkC4ZK/EgS/cQ=
 github.com/moby/sys/mountinfo v0.1.3 h1:KIrhRO14+AkwKvG/g2yIpNMOUVZ02xNhOw8KY1WsLOI=
 github.com/moby/sys/mountinfo v0.1.3/go.mod h1:w2t2Avltqx8vE7gX5l+QiBKxODu2TX0+Syr3h52Tw4o=
+github.com/moby/sys/mountinfo v0.2.0 h1:HgYSHMWCj8D7w7TE/cQJfWrY6W3TUxs3pwGFyC5qCvE=
+github.com/moby/sys/mountinfo v0.2.0/go.mod h1:rEr8tzG/lsIZHBtN/JjGG+LMYx9eXgW2JI+6q0qou+A=
```

Comment:
have you tried `go mod tidy`? I know it's not always works that is why I'm asking :)

Reply:
I think I ran `make vendor` (which should do all the steps, correct?)

Reply:
Let me jump to my computer and double check

Reply:
You were right; `go mod tidy` did give a diff. Perhaps the `make vendor` target needs to be updated to run `go mod tidy` afterwards (it now runs first)?

Reply:
Interesting. From what I read at https://github.com/golang/go/wiki/Modules it seems that `go mod tidy` can add packages to go.mod, meaning it needs to be run before `go mod vendor`.

So, unless `go mod vendor` can modify `go.mod` and `go.sum` (looks like it does not, but I might be wrong here), there's no need to run `go mod tidy` after it.

Reply:
It's odd that CI was green, and didn't show a diff

Reply:
Took me a while to figure out. Apparently, this thing never worked :facepalm:  

See https://github.com/opencontainers/runc/pull/2593